### ISSUE TITLE
Add accessibility 

### ISSUE THREAD

**Issue by mCodex** [mCodex/react-native-rooster#10]
Need to add accessibility to toast. The VoiceOver (iOS) and Talkback (Android) should be able to detect toast's content and read it to the user.
